pages/common/fping.md

```diff
@@ -0,0 +1,15 @@
+# fping
+
+> A more powerful ping which can ping multiple hosts.
+
+- List alive hosts within a given subnet using a netmask:
+
+`fping -a -g 192.168.1.0/24`
```

Comment:
Yes, `g` for generate ! @agnivade 

Actually I have thought including generate in the description but I didn't get a comfortable way,
So, How do think the below

- List alive hosts within a **generated** subnet using a netmask